app/views/idv/session_errors/warning.html.erb

```diff
@@ -13,5 +13,5 @@
 <% end %>
 
 <%= render PageFooterComponent.new do %>
-  <%= link_to t('links.cancel'), idv_cancel_path %>
+  <%= link_to t('links.cancel'), idv_cancel_path(step: :session, location: :warning) %>
```

Comment:
@dawei-nava It seems odd to me to put "session" as the step.  I didn't see session being used as the step elsewhere in the codebase.  Could you please walk me through the decision to put session as the step?  Thanks.

Reply:
@eileen-nava , this is a general session/auth landing page, like from partner agencies. It's more like a general function area about whether user has a valid session or not. Technically user can be redirected to here from any where?

What do you suggest to use?

Reply:
@dawei-nava Ahhhh, okay, that's helpful, thanks.  I was confused because I expected the step to be a specific point in the user flow, like "verify info."

Would it be possible to put where the user is redirected from?  If that isn't straightforward, I think "invalid_session" works.

Thanks for clarifying. 🙏🏻

Reply:
@eileen-nava, changed.